app/views/users/login_user.erb

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
same here. you should not have all of the HTML tags and links on each individual erb